lib/src/block_syntaxes/list_syntax.dart

```diff
@@ -14,6 +14,14 @@ class ListItem {
   final List<String> lines;
 }
 
+/// Invisible string used to placehold for a *Unchecked* CheckBox ('CB').
+/// The character is unicode ZERO WIDTH SPACE (U+200B).
+const invisibleIndicatorForUncheckedCB = '\u{200B}';
```

Comment:
Hmm, I think the `CB` at the end is not a well known acronym [[1]]. To keep the name not too long, how about `indicatorForUcheckedCheckbox`? We don't have to include the word "invisible."

[1]: https://dart.dev/guides/language/effective-dart/documentation#avoid-abbreviations-and-acronyms-unless-they-are-obvious